documentation/asciidoc/computers/configuration/device-tree.adoc

```diff
@@ -850,18 +854,21 @@ This bit field, which describes the features supported by the current bootloader
 | xref:remote-access.adoc#network-boot-your-raspberry-pi[Network boot]
 
 | 2
-| xref:raspberry-pi.adoc#fail-safe-os-updates-tryboot[TRYBOOT]
+| xref:raspberry-pi.adoc#fail-safe-os-updates-tryboot[TRYBOOT_A_B] mode.
 
 | 3
-| xref:raspberry-pi.adoc#usb-mass-storage-boot[USB boot] using the BCM2711 USB host controller.
+| xref:raspberry-pi.adoc#fail-safe-os-updates-tryboot[TRYBOOT]
```

Comment:
Wow, have the bit-positions actually changed, or was the original documentation incorrect?

Reply:
The original documentation was wrong